sound/soc/sof/sof-spi-dev.c

```diff
@@ -33,7 +34,7 @@ static struct snd_soc_acpi_mach spi_machines[] = {
 	},
 };
 
-static const struct sof_dev_desc spi_desc = {
+static const struct sof_dev_desc sue_desc = {
```

Comment:
Why do you want to rename this? It is true, that *currently* only Sue Creek is supported over SPI, and this configuration is probably rather rare, so we might never get a second SPI board, but wouldn't it be better to keep naming consistent? Either keep both these files in the "spi" namespace or make them completely sue-creek specific?

Reply:
"we might never get a second SPI" is this the reason? 
If yes, I could revert it. And I have to set core number , and I only know there are 2 cores on sue-creek, not all spi platform with 2 cores. So I have to do it.

Reply:
@lyakh  yes, I check the namespace again. It is more suitable use spi than sue now because all related files and data structures are in the namespace of spi.